ocaml/client_records/records.ml

```diff
@@ -1057,6 +1057,11 @@ let host_record rpc session_id host =
 		let patch_uuids = List.map (fun x -> Client.Pool_patch.get_uuid ~rpc ~session_id ~self:x) patch_refs in
 		patch_uuids
 	in
+	let get_updates () =
```

Comment:
This can be written in more compact form as follows:

    let get_updates () =
       (x ()).API.host_updates
       |> List.map (fun self -> Client.Pool_update.get_uuid ~rpc ~session_id ~self)